unity/Holo/Assets/Editor/ModelImport/FileImporter.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+using ModelImport.VTKImport;
+
+namespace ModelImport
+{
+    //A class for loading a specific file. Currently STL and specific VTK formats are supported.
+    public class FileImporter
+    {
+        private STLImporter sTLImporter;
+        private VTKImporter vtkImporter;
+        private string fileExtension;
+
+        public Vector3[] BaseVertices { get; private set; }
+        public Vector3[] Vertices { get; private set; }
+        public Vector3[] Normals { get; private set; }
+        public Vector3[] DeltaTangents { get; private set; }
+        public Vector4[] BaseTangents { get; private set; }
+        public int[] Indices { get; private set; }
+        public int IndicesInFacet { get; private set; }
+        public Dictionary<string, Vector3> BoundingVertices { get; private set; } = new Dictionary<string, Vector3>()
+        { { "minVertex", new Vector3()},
+          { "maxVertex", new Vector3()}
+        };
+        private bool dataflow = false;
+
+        //A constructor ensuring FileImporter is format-specific (only STL and VTK for now)
+        public FileImporter(string extension, bool Isdataflow)

```

Comment:
`Isdataflow` should be written with camel case

Reply:
Changed to `dataflow`, while also erased dataflow attribute altogether.